tests/test_processing_common.py

```diff
@@ -50,6 +55,9 @@ def prepare_image_inputs():
 @require_vision
 class ProcessorTesterMixin:
     processor_class = None
+    text_data_arg_name = "input_ids"
+    images_data_arg_name = "pixel_values"
+    videos_data_arg_name = "pixel_values_videos"
```

Comment:
I'm not sure if it adds clarity to have these as class attributes rather than keeping the names around (I had to scroll up to figure out what it was). Nit though, it's fine

Reply:
It comes in handy when a model uses different names, such as `"decoder_input_ids"` for `text_data_arg_name`. Then we can just change those attribute in the processor tester class instead of rewriting all the tests

Reply:
I agree looking through the tests it's a bit confusing (I also had to scroll up to find) but like that it makes it more easily configurable. 

Perhaps we could think of a different name? `"data_arg_name"` isn't obvious what it is. Maybe e.g.  `model_input_text` or `text_input_name`?